pyccel/parser/semantic.py

```diff
@@ -3658,19 +3659,6 @@ def _visit_FunctionCall(self, expr):
             d_methods = cls.methods_as_dict
             method = d_methods.pop('__init__', None)
```

Comment:
Could we call this variable `init_method` or just `init`? The name `method` is a bit confusing 😕

Reply:
Done [615c52d](https://github.com/pyccel/pyccel/pull/2467/commits/615c52d27eefeeec4121b13176dbb19baa2070f6)